packages/requisitions/src/SupplierRequisition/DetailView/reducer.ts

```diff
@@ -17,21 +16,37 @@ export interface SupplierRequisitionStateShape {
   sortBy: SortBy<SupplierRequisitionLine>;
 }
 
+const RequisitionActionCreator = {
```

Comment:
I had to add some imports to this file for the update to work - is there a missing commit?

Reply:
ah nuts sorry about that